sky/serve/replica_managers.py

```diff
@@ -799,8 +866,8 @@ def _probe_all_replicas(self) -> None:
                 replica_to_probe.append(
                     f'replica_{info.replica_id}(url={info.url})')
                 probe_futures.append(
-                    pool.apply_async(info.probe,
-                                     (self._readiness_path, self._post_data)))
+                    pool.apply_async(info.probe, (self._get_readiness_path(
+                        info.version), self._get_post_data(info.version))))
```

Comment:
```suggestion
                    pool.apply_async(info.probe, (self._get_readiness_path(
                        info.version), self._get_post_data(info.version)),),)
```
add several comma here and re-run the formatter (which will make those argument in multiple lines) for readability